src/redist/targets/GenerateBundledVersions.targets

```diff
@@ -1246,6 +1246,52 @@ Copyright (c) .NET Foundation. All rights reserved.
                       Overwrite="true" />
   </Target>
 
+  <Target Name="GenerateBundledMSBuildProps" DependsOnTargets="SetupBundledComponents">
+    <PropertyGroup>
+      <BundledMSBuildPropsFileName>Microsoft.NETCoreSdk.BundledMSBuildInformation.props</BundledMSBuildPropsFileName>
+      <MinimumMSBuildVersionFile Condition="$([System.Text.RegularExpressions.Regex]::Match(%(SDKInternalFiles.Identity),'.*minimumMSBuildVersion').Success) ">%(SDKInternalFiles.Identity)</MinimumMSBuildVersionFile>
+      <BundledMSBuildVersion>$(MSBuildVersion)</BundledMSBuildVersion>
+    </PropertyGroup>
+
+    <ReadLinesFromFile File="$(MinimumMSBuildVersionFile)">
+      <Output TaskParameter="Lines" PropertyName="MinimumMSBuildVersion"/>
+    </ReadLinesFromFile>
+
+    <PropertyGroup>
+      <_BundledMSBuildVersionMajorMinor>$([System.Version]::Parse('$(BundledMSBuildVersion)').ToString(2))</_BundledMSBuildVersionMajorMinor>
+    </PropertyGroup>
+
+    <PropertyGroup>
+      <BundledMSBuildPropsFileContent>
+<![CDATA[
+<!--
+***********************************************************************************************
+$(BundledMSBuildPropsFileName)
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (c) .NET Foundation. All rights reserved.
+***********************************************************************************************
+-->
+<Project>
+  <PropertyGroup>
+    <MinimumMSBuildVersion>$(MinimumMSBuildVersion)</MinimumMSBuildVersion>
+    <BundledMSBuildVersion>$(BundledMSBuildVersion)</BundledMSBuildVersion>
+    <_MSBuildVersionMajorMinor>%24([System.Version]::Parse('%24(MSBuildVersion)').ToString(2))</_MSBuildVersionMajorMinor>
+    <_IsDisjointMSBuildVersion>%24([MSBuild]::VersionGreaterThan('%24(_MSBuildVersionMajorMinor)', '$(_BundledMSBuildVersionMajorMinor)'))</_IsDisjointMSBuildVersion>
```

Comment:
Finally, the comparison can 'bake in' the right hand value as a minor nod to perf.